nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-site-to-site/src/main/java/org/apache/nifi/remote/protocol/http/StandardHttpFlowFileServerProtocol.java

```diff
@@ -68,7 +66,7 @@ public FlowFileCodec getPreNegotiatedCodec() {
     }
 
     @Override
-    protected HandshakeProperties doHandshake(Peer peer) throws IOException, HandshakeException {
```

Comment:
another spot where we're changing the API/exception handling that we should back out for now in my opinion

Reply:
Ack.